contents/docs/libraries/js/index.mdx

```diff
@@ -458,9 +458,9 @@ By default, we store all this information in a `cookie`, which means PostHog can
 
 If you want to change how PostHog stores this information, you can do so with the `persistence` configuration option:
 
-- `persistence: "cookie"` (default): Stores all data in a cookie.
+- `persistence: "localStorage+cookie"` (default): Stores user's distinct ID in a cookie and everything else in the browser's <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage" target="_blank">`localStorage`</a>.
```

Comment:
looking on posthog.com

we have `{"distinct_id":"PqMBBFBfKo638PwvjCWTSZ6dnswYzJ3RtH8VIHCBWAy","$sesid":[1713543186224,"018ef721-f332-7f12-85d0-9def8ccfd0fd",1713543181106],"$epp":true}`

so, it's 

 * distinct id, 
 * session id (with start time and last activity time) 
 * and $epp 🤷

Reply:
ah `export declare const ENABLE_PERSON_PROCESSING = "$epp";`